plugins/kubernetes/src/kubernetes-auth-provider/ServerSideAuthProvider.ts

```diff
@@ -17,13 +17,18 @@
 import { KubernetesAuthProvider } from './types';
 import { KubernetesRequestBody } from '@backstage/plugin-kubernetes-common';
 
-export class GoogleServiceAccountAuthProvider
+/**
+ * No-op KubernetesAuthProvider, authorization will be handled in the kubernetes-backend plugin
+ *
+ * @alpha
```

Comment:
Is this supposed to be alpha? Seems like something pretty stable to me tbh 😅

Reply:
I'm too nervous to cut a v1 😂